src/test/java/org/minions/devfund/richard/whackamole/WhackAMoleTest.java

```diff
@@ -0,0 +1,162 @@
+package org.minions.devfund.richard.whackamole;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class.
+ */
+public class WhackAMoleTest {
+
+    private WhackAMole myWhackAMole;
+    private static final int ONE_NE = -1;
+    private static final int ONE = 1;
+    private static final int THREE = 3;
+    private static final int FIVE = 5;
+    private static final String GRID = "* * * * * ";
```

Comment:
Don't create this constants.
Please create final local variables inside test method.

Reply:
Ok I will do that.

Reply:
About private static final String GRID = "* * * * * ";
Intellij tell that we should use as constat.

![image](https://user-images.githubusercontent.com/28739313/44003602-f1a8970a-9e22-11e8-9571-a53421cae8bc.png)